crates/noirc_frontend/src/lexer/token.rs

```diff
@@ -325,7 +325,7 @@ pub enum Attribute {
     Builtin(String),
     Oracle(String),
     Deprecated(Option<String>),
-    Test(/* should fail: */ bool),
+    Test(/* expect failure: */ bool),
```

Comment:
This is still a comment. Please define a struct variant or separate into two variants.